packages/video-feature/types.ts

```diff
@@ -0,0 +1,12 @@
+import type { AuthorBylineProps } from 'packages/author-byline/types'
+import type { IntroProps } from '@hashicorp/react-intro/types'
+import type { InlineVideoProps } from '@hashicorp/react-inline-video/types'
+
+export interface VideoFeatureProps {
+  appearance?: 'light' | 'dark'
+  contentPosition?: 'left' | 'right'
+  heading: IntroProps['heading']
+  description: IntroProps['description']
+  author?: AuthorBylineProps
+  video: InlineVideoProps
```

Comment:
Should we be picking the props we need here vs passing all of them along?

Wouldn'nt assume we expose appearance for author or video as those come from video.

Video also does not need to expose the gradientPosition as that is inferred from the contentPosition etc.

Reply:
Good points all around! I've used `Pick` instead to more specifically assign the type here:
https://github.com/hashicorp/react-components/pull/784/commits/98fe500d8415fff17d0f358ebb77b764e814ba0d